modules/codelite/codelite_project.lua

```diff
@@ -268,9 +267,21 @@
 		local pauseexec  = iif(prj.kind == "ConsoleApp", "yes", "no")
 		local isguiprogram = iif(prj.kind == "WindowedApp", "yes", "no")
 		local isenabled  = iif(cfg.flags.ExcludeFromBuild, "no", "yes")
-
-		_x(3, '<General OutputFile="%s" IntermediateDirectory="%s" Command="%s" CommandArguments="%s" UseSeparateDebugArgs="%s" DebugArguments="%s" WorkingDirectory="%s" PauseExecWhenProcTerminates="%s" IsGUIProgram="%s" IsEnabled="%s"/>',
-			targetname, objdir, command, cmdargs, useseparatedebugargs, debugargs, workingdir, pauseexec, isguiprogram, isenabled)
+    local ldPath = ''
```

Comment:
Somehow I managed to not notice this before, the indentation here isn't right. It looks like you're using spaces while the rest of the file is using tabs.

Reply:
Ah, I never use tabs.

Also the LD_LIBRARY_PATH may be linux specific. So I will have to figure out how to put the correct value in there for windows.